test/testPaint.cpp

```diff
@@ -202,6 +202,110 @@ TEST_CASE("Bounding Box", "[tvgPaint]")
     Initializer::term();
 }
 
+TEST_CASE("Text Bounding Box", "[tvgPaint]")
```

Comment:
Let's merge this with exsiting `TEST_CASE("Bounding Box", "[tvgPaint]")` No need to separate them.

Reply:
Since the bounding box results inevitably differ, I organized them into sections within a single test case. Is that alright?